### PR TITLE
Update rubocop version and rectify related test failures.

### PR DESCRIPTION
**Pull request by websitescenes**

### CODE REVIEW

**Comment by jerearista** [comment]
retest this please after removing the Gemfile.lock in CI...

**Comment by jerearista** [comment]
retest this please (updated CI jobs)

**Comment by jerearista** [comment]
retest this please (Verifying updated CI job description)

File: Gemfile [hunk 1]
```diff
@@ -6,6 +6,7 @@ gem 'net_http_unix'
 gem 'netaddr'
 
 group :development do
+  gem 'rubocop', '0.34.2'
```
Comment (devrobo) [inline Gemfile]: Why not 0.35.1?

Comment (websitescenes) [inline Gemfile]: Good catch. I was going off my installed version thinking it was the latest. This version was released in the last 7 days. Probably should update.

**Comment by websitescenes** [comment]
retest this please

**Pull request merged**